lib/services/local/access_list.go

```diff
@@ -172,6 +173,26 @@ func (a *AccessListService) ListAccessLists(ctx context.Context, pageSize int, n
 	return a.service.ListResources(ctx, pageSize, nextToken)
 }
 
+// RangeAccessLists returns access list resources within the range [start, end).
+func (a *AccessListService) RangeAccessLists(ctx context.Context, start string, end string, sort *types.SortBy) iter.Seq2[*accesslist.AccessList, error] {
+	// Currently, the backend only sorts on lexicographical keys and not
+	// based on fields within a resource
+	if sort != nil && (sort.Field != "name" || sort.IsDesc != false) {
+		return func(yield func(*accesslist.AccessList, error) bool) {
+			yield(nil, trace.BadParameter("unsupported sort, only name:asc is supported, but got %q (desc = %t)", sort.Field, sort.IsDesc))
```

Comment:
How is the caller going to handle this? Don't we just blank out the page to display an error, in the web UI? It would be nice if we could use some error that specifically signals "advanced filtering and sorting is unavailable" rather than just `BadParameter`.